app.py

```diff
@@ -1,46 +1,48 @@
-from flask import Flask, render_template, request, flash, redirect, url_for, jsonify
-from flask_cors import CORS
+from flask import Flask, render_template, request, flash, redirect, url_for, jsonify, g
+# from flask_cors import CORS
 from hashids import Hashids
 import sqlite3
-hashids = Hashids()
+
+
+def get_db_connection():
+    db = sqlite3.connect('database.db')
+    db.row_factory = sqlite3.Row
+    return db
 
 
 
 app = Flask(__name__)
-CORS(app)
+# CORS(app)
 
 app.config['SECRET_KEY'] = 'this should be a secret random string'
 
 hashids = Hashids(min_length=5, salt=app.config['SECRET_KEY'])
-hashid = hashids.encode(8)
-print(hashid)
+# hashid = hashids.encode(8)
+# print(hashid)
 
-def get_db_connection():
-    connection = sqlite3.connect('database.db')
-    connection.row_factory = sqlite3.Row
-    return connection
 
 
-@app.route('/', methods=['GET', 'POST'])
```

Comment:
missing dynamic route - something like </:id> which can render the short url - think this is party why its not working properly!